### ISSUE TITLE
Can't specify a validity time range for projects

### ISSUE THREAD

**Issue by jgidi** [jgidi/webpage-plasmas-udec#10]
The notion of a project as implemented by Wowchemy is not the same as we use it. This raises, for example, the issue of projects only accepting one publication date and not being able to handle a project's validity time range.

For the previous reason I was thinking that we could re-implement a project type matching our use case. However, an easier solution would be to just add two parameters `start_date` and `end_date` to the project's front-matter and modify the way that projects are listed such that this is shown. Also, we could use `end_date` to filter current or past projects, as proposed in https://github.com/jgidi/webpage-plasmas-udec/issues/9#issuecomment-828753235.

**Comment by rnavarromal** [comment]
I believe projects must have these properties:
- `Title`
- `Authors`
- `project ID` (the unique ID number of the project)
- `Program` (e.g. "fondecyt", "PhD scholarship", "postdoc", etc.)
- `year` (in the form "2018" to show the starting date only, or "2018-2021" to show a range; or maybe two different properties `start_date` and `end_date`, with `end_date` optional)
- URL (pointing to an ANID repository e.g. http://repositorio.conicyt.cl/handle/10533/162025)

Since some publications can be associated to a list of projects, then **the project's page should list all associated publications** (as it is currently done with the members' page).

**Comment by jgidi** [comment]
Totally agree with your comment. 

I'm thinking that a good way to implement it would be to just add those properties in the project's frontmatter, and to add a layout to `layouts/project/single.html`, which will be used to render the page of each single project.

With respect to the dates, we could keep the 'date' property as it is expected by hugo, and to use a second optional key 'date_end'. Also, those dates should specify a day and month to make them compliant with the native hugo date handling. Later, in the project layout, we can specify to print only the year of the dates, and to show them as a range if 'date_end' is defined.

As for showing the associated publications for each projects page, that is easy to implement by using the projects title or id (i'd go for the second) defined in the publication frontmatter as a filter to show them on the projects layout.

In summary, we can implement all of what you proposed by just adding editing the default single project layout, and adding a corresponding key in the frontmatter of the projects and publications

If you agree, I can start working on it soon enough!

**Comment by rnavarromal** [comment]
go ahead

**Comment by jgidi** [comment]
Closing issue addressed by PR #12.
